src/main/java/ch/njol/skript/tests/runner/EffAssert.java

```diff
@@ -100,7 +100,10 @@ public TriggerItem walk(Event e) {
 	}
 
 	@Override
-	public String toString(@Nullable Event e, boolean debug) {
-		return "assert " + condition.toString(e, debug);
+	public String toString(@Nullable Event event, boolean debug) {
+		if (event == null || condition == null)
```

Comment:
shouldn't this only be if condition is null? it is valid to pass a null event to the to string of a condition